### PR TITLE
Bug 1273416 - [TV][Home] Press down arrow key while focusing on Done …

### PR DESCRIPTION
**Pull request by luke-chang**
…button in card selecion page will move focus to previous card

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c4234dc8ac89c142edc1ea5ae3cc75e4ce99b898

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3f5e16bc38a76f3e453c678b16bfdcf4a4343c7b

**Comment by rexboy7** [comment]
Please change your commit message.

File: tv_apps/smart-home/js/card_picker.js [hunk 1]
```diff
@@ -90,25 +90,17 @@
       elem.focus();
       if (elem.classList.contains('app-button')) {
         this._scrollTo(elem);
-        this._lastFocusedAppButton = elem;
+        this.input.dataset.navDown =
+          this.hideCardPickerButton.dataset.navUp =
+          '#card-picker-grid-view [data-card-id="' + elem.dataset.cardId + '"]';
```
Comment (rexboy7) [inline tv_apps/smart-home/js/card_picker.js]: A little bit hacky :-/

**Pull request merged**